modules/ROOT/partials/configuration/format_noneditable_selector.adoc

```diff
@@ -0,0 +1,22 @@
+[[format_noneditable_selector]]
+== `+format_noneditable_selector+`
+
+This option allows a non-editable (`+contenteditable="false"+`) element that matches the selector specified to be wrapped with a format.
+
+Formats that can wrap include block formats that have the `+wrapper+` parameter set to `+true+` and inline formats. Selector formats are not supported.
+
+In addition to a non-editable element matching the selector, any non-editable element that has the `+data-mce-cef-wrappable+` attribute can be wrapped with a format.
```

Comment:
The `format` mentioned here for the `data-mce-cef-wrappable` ... is that limited by the statement above, or can *all* formats wrap non-editable elements with `data-mce-cef-wrappable`? I assume it's still limited, but I'm not sure if that's clear.

Reply:
Interesting thought this was internal. From what I can see there is no mention of this new attribute in the changelog. It only mentions the selector. We have not documented any other internal attributes from what I know. But this one doesn't seem to be stripped by default on save so I guess this is a special case. So is the intention that we want this to be public?

- [ ] Confirm that this attribute is to be public knowledge

Reply:
I have flipped the paragraphs around which hopefully makes it clearer.

With regards to whether to mention `data-mce-cef-wrappable` or not. I think the fact that like you say it is not ephemeral and also is something that is always enforced i.e not integrator configurable, it is probably good for people to be aware of it in some capacity.

It probably should have been mentioned in the changelog but wasn't.